config/puma.rb

```diff
@@ -12,3 +12,10 @@
 end
 
 plugin :tmp_restart
+
+before_fork do
+  require 'puma_worker_killer'
+
+  rolling_restart_interval = (ENV['PUMA_WORKER_RESTART_INTERVAL'] || (6 * 3600)).to_i
```

Comment:
It's may be better if PUMA_WORKER_RESTART_INTERVAL is nil, disable puma worker killer.